consensus/service/config/src/lib.rs

```diff
@@ -248,6 +253,7 @@ mod tests {
     #[test]
     fn test_deployed_uris_with_pubkey() {
         let config = Config {
+            network_id: "local".to_string(),
```

Comment:
I don't understand, NETWORKNAME is not a variable in our code so there is no way to implement a test like that.

NETWORKNAME is a variable used in the kubernetes manifests. It seems quite likely that this could be used for the network-id by the deployment code. But exactly how this is set is up to ops and not a concern of this PR.

Reply:
Sorry, you're right, this comment doesn't make sense.